benchmark/bipedal-timings.cpp

```diff
@@ -26,6 +26,7 @@
 #include "crocoddyl/multibody/costs/state.hpp"
 #include "crocoddyl/multibody/costs/control.hpp"
 #include "crocoddyl/multibody/actuations/full.hpp"
+#include "crocoddyl/multibody/actuations/floating-base.hpp"
```

Comment:
It weird that this file doesn't compile for you.
If you pay attention, we need to include the `crocoddyl::ActuationModelFloatingBase`.

Reply:
Hi @cmastalli, this file compiles fine if I include the floating base header in bibedal-timings.cpp. This `#include "crocoddyl/multibody/actuations/floating-base.hpp"` was absent from bipedal-timings.cpp previously.  
In more detail:
Previously, this include was in the `DifferentialActionModelContactFwdDynamics` file so when I modified the files include/crocoddyl/multibody/actions/contact-fwddyn, I replaced the floating base header by the abstract actuation model, causing the bidebal-timings.cpp to fail to compile. So I added the incude flag in bipedal-timings.cpp
Is this fine ?

Reply:
Ok, I see.
Sorry I didn't realize it.

Thanks for the explanation